### DISCUSSION TITLE
Prisma Issue with TypedSQL and Custom Output Configuration

### DISCUSSION THREAD

**Discussion by monchansuzuki** [prisma/prisma#25716] [Q&A]
### Question

I would like to use the amazing TypedSQL feature to write type-safe queries. However, I'm currently facing an issue because I'm using a custom output configuration.

When I tried to generate the --sql and import the query into my TypeScript file, it didn't seem to work as expected. The SQL query is located in /prisma/sql, as mentioned in the documentation.

Here’s my schema.prisma file for reference:

### Information about Prisma Schema, Client Queries and Environment (optional)

generator client {
  provider        = "prisma-client-js"
  output          = "./prisma-client"
  binaryTargets   = ["native", "debian-openssl-3.0.x", "debian-openssl-1.1.x"]
  previewFeatures = ["typedSql"]
}

**Comment by ludralph** [comment]
Hi @monchansuzuki 

Can you please share any error messages you're getting? Also, can you provide the relevant part of the schema file and the schema file for reproduction?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
